modules/fluid_properties/src/userobjects/IdealGasFluidProperties.C

```diff
@@ -35,7 +35,9 @@ IdealGasFluidProperties::pressure(Real v, Real u) const
   if (v == 0.0)
     mooseError(name() << ": Invalid value of specific volume detected (v = " << v << ").");
 
-  return (_gamma - 1.) * u / v;
+  // The std::max function serves as a hard limiter, which will guarantee non-negative pressure
+  // when resolving strongly nonlinear waves
+  return std::max(1e-8, (_gamma - 1.) * u / v);
```

Comment:
It does not have really any meaning, so I do not think I would make it a constant. It is just a SWAG...